scripts/linters/general_purpose_linter.py

```diff
@@ -514,90 +456,18 @@
         'excluded_files': (),
```

Comment:
Can we move the input and map function check (line 448) to disallowed-functions-and-replacements?

Reply:
Done!